### ISSUE TITLE
Error: Object 'networking' is unknown, try 'nmcli help'

### ISSUE THREAD

**Issue by IMG-PRCSNG** [msolters/wifi-control-node#15]
I am currently using _nmcli tool, version 0.9.8.8_

I am really not sure but there seems to be a significant change in the constants and command objects used in this version.
- I am not able to find "networking" object in the nmcli command.
- The scan result doesn't seem to have CHAN attribute when i run `nmcli -m multiline device wifi list`

Can anyone verify and provide what might be the cause for discrepancy?

**Comment by IMG-PRCSNG** [comment]
Looks like #7 is partially solving this issue. This must be due to the version discrepancies of nmcli - 14.04 LTS doesn't ship with nmcli version where networking object was introduced . Will update if i manage to port the code for lower versions of nmcli.

**Edit**
Actually at the end of #7 conversation, i found a fork by @cmaissan - [here](https://github.com/cmaissan/wifi-control-node) with a branch that actually fixes it for version 0.9.8.8.

**Comment by msolters** [comment]
Right, and as I stated once in #7 , if this were to be rewritten using anything it should be `iw`

**Comment by msolters** [comment]
@IMG-PRCSNG I rewrote the package in such a way as to remove any reference to `nmcli networking`.  I'd be very interested to know how it works for you.  You can checkout the branch over at #16 .
